igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -69,6 +69,17 @@ class TravisService implements BuildService{
         accounts
     }
 
+    void setAccounts() {
+        this.accounts = getAccounts()
+    }
+
+    void syncRepos() {
+        Response response = travisClient.usersSync(getAccessToken())
+        if (response.status != 200) {
```

Comment:
@lwander I checked retrofit, I could not find a function like that. I changed the test to be >= 400. That should cover the errors.